presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -118,6 +118,7 @@
     private int fileSystemMaxCacheSize = 1000;
 
     private boolean writesToNonManagedTablesEnabled;
+    private boolean createNonManagedTableEnabled = true;
```

Comment:
If you decided to rename a configuration name please rename all the variable names as well.